ESSArch_Core/frontend/static/frontend/scripts/controllers/BaseCtrl.js

```diff
@@ -1152,6 +1152,20 @@ export default class BaseCtrl {
       }
     };
 
+    vm.multipleIpResponsible = function() {
+      if ($scope.ips.length > 0) {
+        var responsible = true;
+        $scope.ips.forEach(function(ip) {
+          if (ip.responsible.id !== $rootScope.auth.id) {
+            responsible = false;
+          }
+        });
+        return responsible;
```

Comment:
Do we need the `responsible` variable? Can't we just return the bool as soon we find a differing responsible id? 

```
        $scope.ips.forEach(function(ip) {
          if (ip.responsible.id !== $rootScope.auth.id) {
            return false;
          }
        });
        return true;
```

Reply:
forEach has no return value, so your example should always return true. So we do need the extra variable